### DISCUSSION TITLE
Why all the ⚠️? (Everything stuck on "Waiting to back up...")

### DISCUSSION THREAD

**Discussion by pepper-lees** [seedvault-app/seedvault#546] [Q&A]
Hi,

I'm running LineageOS 20.0 and I decided to try out Seedvault since I saw the option in the OS backup menu. I set up with my Nextcloud, but everything is stuck on "Waiting to back up". In the status page, everything has a ⚠️ symbol, but no info appears when I tap on it. When I long press, it just takes me to the given app's info page in my phone's settings. Also, the "Backup now" button is disabled.

What's going on? Is there some error happening, and if so, how do I see what the error is? Could it be caused by a timeout with my Nextcloud server? It *is* online and responsive, but it is very slow. 
![Screenshot_20230712-210352_Seedvault](https://github.com/seedvault-app/seedvault/assets/11487870/d233dde2-2455-4d2b-91ff-48c566264f7a)
![Screenshot_20230712-210359_Seedvault](https://github.com/seedvault-app/seedvault/assets/11487870/9aa4b5f3-6152-4a1b-9977-58215ec9498b)

**Comment by pepper-lees** [comment]
is it ok to bump discussions? I'm still wondering about this.

**Comment by grote** [comment]
Is nothing shown as backed up? Not a single app for more than a day? If that's the case, maybe the system isn't scheduling backups for some reason. Did you try a manual backup? Does the app list look the same afterwards?

**Reply by pepper-lees** [reply]
Hmm. Last night, that was the case (nothing was shown as backed up, and everything showed a ⚠️, even though I manually triggered a backup multiple times). But then I decided to disable the "App backup" toggle and manually triggered another backup. Now, _most_ apps have a green check. Some apps have a ⚠️ but are labelled "Not backed up as it wasn't used recently" or "App reported no data for backup" or "backup quota exceeded". The only items left on "Waiting to back up..." are _Seek_, _Whatsapp_, _Yet Another SafetyNet Attestation Checker_, and _Local contacts_.

Btw, I just checked the "About" tab to see what version I have installed, and it says `13.3.3`. But on github, the releases only go up to 11-2.3. What's the deal there?

**Reply by pepper-lees** [reply]
Just noticed a notification from 2 days ago saying "Backup failed". Is there somewhere I can find a debug log that has more info on what caused the backup to fail earlier?

**Reply by grote** [reply]
ok cool, seems like things are working now. Note that backing up to Nextcloud can be flaky and cause backups to fail.

@chirayudesai maybe we should remove releases from the project as it seems to confuse people?

**Reply by chirayudesai** [reply]
> maybe we should remove releases from the project as it seems to confuse people?

Done.

**Comment by ADogNamedBoo** [comment]
Sorry, I do not have a solution, but a similar problem: I work with /e/ and seevault 13-3.3 and tried to start a seevault backup on an empty usb-C stick. It starts, runs till approximately 12% and then tells that the backup failed.
The stick now contains a folder named .SeedVaultAndroidBackup with a subfolder 1725005093917 (there are 38 objects including the file .backup.metadata) and a file .nomedia
The backup status of the apps look like above: all apps that can be backuped are waiting to backup.
Now the stick is not in the phone.
I tried before to do backups like described and always ended up like this. For the last test I deleted the stick and started again.

**Reply by grote** [reply]
Could you try a different flash drive or run a check program on the existing one? May be faulty.

**Reply by ADogNamedBoo** [reply]
The flash drive is new, from SanDisk - no no name product - bought for Seevault backups. After failing there was plenty of unused space on it. Looking at it from a Linux system with its usb-a plug was possible.
I ran a check program on it with "badblocks" tool on the Linux system with alot of writing tests which did not find a problem. Afterwards the flash drive was empty, I formatted it again with the phone. Seevault set it up again and started the backup: It went like before: No problems from 1% to 7%, then it jumped to 10%, then back to 8%, 9%. Back to 10% the message "Sicherung fehlgeschlagen"/backup failed showed.

**Reply by grote** [reply]
Seedvault 13-3.3 is pretty old. You may also run into old issues like https://github.com/seedvault-app/seedvault/issues/491 but without a log hard to tell.
